client/consensus/manual-seal/src/consensus.rs

```diff
@@ -33,6 +33,8 @@ pub trait ConsensusDataProvider<B: BlockT>: Send + Sync {
 	/// Block import transaction type
 	type Transaction;
 
+	type Proof: Send + Sync + 'static;
```

Comment:
```suggestion
	/// The proof type.
	type Proof;
```
Bounds not strictly required here